src/module/actor/character/sheet.ts

```diff
@@ -781,14 +781,100 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
 
         navTitleArea.innerText = game.i18n.localize(activeTab.dataset.tooltip ?? "");
         const manageTabsAnchor = htmlQuery<HTMLAnchorElement>(sheetNavigation, ":scope > a[data-action=manage-tabs]");
-        if (manageTabsAnchor) new PCSheetTabManager(this.actor, manageTabsAnchor);
+        manageTabsAnchor?.addEventListener("click", () => {
+            this.#manageTabs(manageTabsAnchor);
+        });
 
         sheetNavigation.addEventListener("click", (event) => {
             const anchor = htmlClosest(event.target, "a[data-tab]");
             if (anchor) navTitleArea.innerText = game.i18n.localize(anchor.dataset.tooltip ?? "");
         });
     }
 
+    async #manageTabs(anchor: HTMLElement) {
+        const actor = this.actor;
+        const tabVisibility: Record<string, boolean> = actor.flags.pf2e.sheetTabs;
+        const tabs = [
+            {
+                label: "PF2E.TabCharacterLabel",
+                name: "character",
+            },
+            {
+                label: "PF2E.TabActionsLabel",
+                name: "actions",
+            },
+            {
+                label: "PF2E.TabInventoryLabel",
+                name: "inventory",
+            },
+            {
+                label: "PF2E.TabSpellbookLabel",
+                name: "spellcasting",
+            },
+            {
+                label: "PF2E.TabCraftingLabel",
+                name: "crafting",
+            },
+            {
+                label: "PF2E.TabSkillsLabel",
+                name: "proficiencies",
+            },
+            {
+                label: "PF2E.Item.Feat.Plural",
+                name: "feats",
+            },
+            {
+                label: "PF2E.Item.Effect.Plural",
+                name: "effects",
+            },
+            {
+                label: "PF2E.Biography",
+                name: "biography",
+            },
+            {
+                label: "PF2E.TabPathfinderSociety",
+                name: "pfs",
+            },
+        ].map((t) => ({ ...t, checked: !!tabVisibility[t.name] }));
+
+        const content = await fa.handlebars.renderTemplate("systems/pf2e/templates/actors/character/manage-tabs.hbs", {
+            tabs,
+        });
+        const html = createHTMLElement("div", { innerHTML: content }).firstElementChild;
+        if (!(html instanceof HTMLElement)) throw new Error("Unexpected result rendering tab manager");
+
+        const checkboxes = html.querySelectorAll("input");
+        html.addEventListener("change", async (evt) => {
+            const checkbox = evt.target;
+            if (!(checkbox instanceof HTMLInputElement)) return;
+
+            const tabName = checkbox?.dataset.tabName;
+            if (!tabName) return;
+
+            for (const c of checkboxes) {
+                c.readOnly = true;
```

Comment:
```suggestion
                c.disabled = true;
```
Checkboxes lack a `readOnly` attribute

Reply:
They probably had css. It looked like it work, I saw the flashing as it updated, but it was probably a lie.